examples/example-project/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E111,E114,E121,E123,E226,W292
```

Comment:
Remove E111 and E114.
We use 4-space indents and should get a warning if that's not done.